managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,339 @@
+## Embedded Workflows
+
+*Embedded Workflows* provide the capability to simplify service catalog creation and improve the overall service catalog experience through a step-based automation system. Each step within a workflow runs a container to complete processes. Workflows are imported from GitHub repository content into {{ site.data.product.title_short }} where they are made available for users to run.
```

Comment:
> Each step within a workflow runs a container to complete processes.

This isn't exactly true, Task steps will run a container but there are other types of steps.  You could say something like "Workflows can run docker containers to accomplish task the user requires as part of the state machine"

Reply:
> Workflows are imported from GitHub repository content

Same comment as above re: GitHub

Reply:
For the first, the sentence is updated to be: Workflows can run docker containers to accomplish a task that the user requires as part of the state machine. 

For the second, this is updated to be just Git repository.